### ISSUE TITLE
Review: NavBar.jsx and RoomDropdown.jsx

### ISSUE THREAD

**Issue by khirano7** [room-directory/room-directory#94]
## Overview
We are going to conduct our third IDPM Review.

## Review Branch
[review-3](https://github.com/room-directory/room-directory/tree/review-3)

## Files to review
[NavBar.jsx](https://github.com/room-directory/room-directory/blob/review-3/app/imports/ui/components/NavBar.jsx)
[RoomDropdown.jsx](https://github.com/room-directory/room-directory/blob/review-3/app/imports/ui/components/RoomDropdown.jsx)

## Checklists
[Architecture](https://github.com/cammoore/campus-cravings/tree/master/checklists/architecture-checklist.md)
[Design](https://github.com/cammoore/campus-cravings/tree/master/checklists/design-checklist.md)
[JS](https://github.com/cammoore/campus-cravings/tree/master/checklists/js-checklist.md)
[React](https://github.com/cammoore/campus-cravings/tree/master/checklists/react-checklist.md)

## Due date

2/13

## For more information
The review process is documented at: http://courses.ics.hawaii.edu/ics414s23/morea/review/reading-idpm-review.html

**Comment by itsjerie** [comment]
### NavBar.jsx
D-05: L96-97, just combine line

### RoomDropdown.jsx
D-05: L11 could just remove?

**Comment by timothyro** [comment]
## NavBar.jsx
- REACT-01: could split how to navigate into its own component
## RoomDropdown.jsx
- add todo or explanation comment

**Comment by khirano7** [comment]
## NavBar.jsx
- Design-05: Add comments describing component
- Add comment describing what still needs to be done (ex: adding site navigation for all roles)
- Maybe combine the navigation to studentRequests and facultyRequests

## RoomDropdown.jsx
- Design-05: Add comments

**Comment by carakaki808** [comment]
NavBar:

- Design-05: add more comments to code. Hard to follow

RoomDropdown:

- Design-05: L11: Elaborate more?

**Comment by KyoufuuN1K1** [comment]
NavBar.jsx
DE-05: Needs more comments to stay organized

RoomDropdon.jsx
DE-05: Maybe add more comments?

**Comment by michnotmeesh** [comment]
NavBar.js

- REACT-06: Define constants in withTracker(). Currently uses useTracker(). But I'm pretty sure that's the same thing

RoomDropdown.js

- REACT-04: Destructure props in component parameter. Looks good actually.
